src/RowOption/RowOptionContainer.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import RowOptionContent from './RowOptionContent';
+import RowOptionLeftIcon from './RowOptionLeftIcon';
+import RowOptionRightButton from './RowOptionRightButton';
+
+import { Touchable } from '../Touchable';
+import { StyleSheet } from '../PlatformStyleSheet';
+
+type Props = {|
+  +border?: 'long' | 'short' | 'shaped',
+  +type: 'localization' | 'destination' | 'airplane' | 'bus' | 'train',
+  +header: string | React.Node,
+  +subheader: string | React.Node,
+  +info?: string | React.Node,
+  +onItemPress: () => void,
+  +onAddPress: () => void,
+|};
+
+export default function RowOptionContainer({
```

Comment:
Can we please name this file and component just `RowOption` to make clear that it is the main one. 
Also we probably should export it with `index.js` like the rest to make it consistent.

Reply:
1.Sure I can rename it. 
2. Basically I didn't export it, because I thought that `RowOption` component will be in the future a part of `PlacePicker` component and will be within `PlacePicker` folder, but I will add `index.js` for it.